packages/devtools_app/lib/src/screens/memory/panes/diff/diff_pane.dart

```diff
@@ -114,6 +114,10 @@ class SnapshotInstanceItemPane extends StatelessWidget {
 
 /// `\v` adds vertical space
 const _snapshotDocumentation = '''
+Find unexpected memory usage by comparing two heap snapshots:
```

Comment:
Nit: maybe "Examine changes in memory usage by comparing two heap snapshots:"? Or "Debug changes in memory usage..."?